PrivateCollections/Ethiopia/MentasnotAbbuhay/MEAG002.xml

```diff
@@ -0,0 +1,404 @@
+                                          <locus from="145" to="160"></locus>
+                                      </item>
+                                      <item xml:id="q12" n="11">
+                                          <num value="11">፲፩</num>
+                                          <dim unit="page">16</dim>
+                                          <locus from="161" to="176"></locus>
+                                          <note>The last three of the sixteen pages are not paginated</note>
+                                      </item>
+                                      <item xml:id="q13" n="B">                  
+                                          <dim unit="page">16</dim>
+                                          <locus from="xvii" to="xxxii"></locus>
+                                      </item>
+                                  </list>                                  
+                              </collation>
+                              <condition key="intact"></condition>
+                          </supportDesc>
+                           <layoutDesc>
+                               <layout columns="1" writtenLines="11">
+                                   <ab type="pricking">Pricking is visible.</ab>
+                                   <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
+                                   <ab type="ruling">Ruling is visible.</ab>
+                                   <ab type="ruling">The upper line is written above the ruling,
+                                       the lower line is written above the ruling.</ab>
+                                   <locus from="v" to="xxviii"></locus>
+                                   <note>The first and the last four pages of the manuscript are not ruled.</note>
+                               </layout>
+            </layoutDesc>
+                   </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Regular hand, the manuscript is copied by <persName ref="PRS14050MentasnotAbbuhay" role="scribe">
+                                </persName>.
+                            </desc>
+                            <seg type="ink">Black, red.</seg>
+                                <date when="2022"/> 
+                            </handNote>
+                        </handDesc> 
+                        
+                        <decoDesc> 
+                            <decoNote type="band" xml:id="d1">
+                                <locus target= "#1 #19 #38 #60 #81 #102 #126 #153"/>
+                                <desc>Decorative Ḥarags are illustrated at the beginning of each new section.</desc>
+                            </decoNote>
+                            
+                            <decoNote type="miniature" xml:id="d2">
```

Comment:
are these really miniatures or drawings? usually amuletic images are not miniatures (coloured, with persons etc)